config/locales/en.yml

```diff
@@ -3095,7 +3095,7 @@ en:
     site:
       edit_tooltip: Edit the map
       edit_disabled_tooltip: Zoom in to edit the map
-      createnote_tooltip: Add a note to the map
+      createnote_tooltip: Move the marker and write a note to fix map errors.
```

Comment:
You are writing about the marker in a tooltip that is shown before the marker is visible.

Reply:
what should be written then?